src/views/virtualmachines/actions/components/VirtualMachineMigration/hooks/useCurrentStorageMigration.ts

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import useVirtualMachineInstanceMigration from '@kubevirt-utils/resources/vmi/hooks/useVirtualMachineInstanceMigration';
+
+const useCurrentStorageMigration = (vm: V1VirtualMachine) => {
+  const [migrationStartedTime] = useState(new Date());
```

Comment:
u sure u need state here?

Reply:
it can be a ref. 
I can't do a `const` outside the hook because it will run when this code is important but not executed.
I need the date when it's executed ( the migration start )